src/errors/context-error.ts

```diff
@@ -4,7 +4,7 @@ export const NO_CONTEXTS_SAVED = `Specify what AsyncAPI file to be used.
 These are your options to specify in the CLI what AsyncAPI file should be used:
  - You can provide a path to the AsyncAPI file: asyncapi <command> path/to/file/asyncapi
  - You can also pass a saved context that points to your AsyncAPI file: asyncapi <command> context-name
- - In case you did not specify a context that you want to use, the CLI checks if there is a default context and uses it. To set default context run: asyncapi context use mycontext
+ - In case you did not specify a context that you want to use, the CLI checks if there is a default context and uses it. To set default context run: asyncapi context use defaultContext
  - In case you did not provide any reference to AsyncAPI file and there is no default context, the CLI detects if in your current working directory you have files like asyncapi.json, asyncapi.yaml, asyncapi.yml. Just rename your file accordingly.
```

Comment:
I think we have to change message completely, Since this error is thrown when there is no context file created, It would be also nice to add steps on how to create context 

Something like this - 


```
No context has been set yet, 
To use a context as current you need to create one, follow these steps - 
 - Create a new context: asyncapi config context add <context-name> <file-path | URL>
 - Set this context as current: asyncapi config context use <context-name>
```

Reply:
@pranay202 play a bit with the CLI to get the idea how these different options work

Reply:
Thank you for your reviews @Souvikns and @derberg. I will surely work on this and let you know with the PR.

Reply:
Can we have it like this -
```
No context has been set yet, 
To use a context as current you need to create one, follow these steps - 
 - Create a new context: asyncapi config context add <context-name> <file-path | URL>
 - Set this context as current: asyncapi config context use <context-name>
 - Try asyncapi --help to learn more about commands or go through the <a href="https://www.asyncapi.com/docs">docs</a>
```

Reply:
Yeah looks fine to me 👍🏼

Reply:
Thank you for your reviews @Souvikns.